### PR TITLE
Deploy fb-av to saas-test namespace

### PR DESCRIPTION
**Pull request by njseeto**
[Trello](https://trello.com/c/MPnUo7jW/2752-autocomplete-editor-fb-av)

### Adjust circle config to deploy into saas-test namespace
With the addition of the autocomplete component, form editors are now able to upload csv files onto the editor app. We need to ensure these uploaded files are not malicious. To do this, we have decided to deploy our current fb-av app into the saas namespace.

### Use namespace variable in eks charts
Since the fb-av app needs to be deployed into both the platform and saas namespaces, we need to ensure the correct values are interopolated.

### Add namespace to deployment.yaml
This is to ensure the app is deployed into the correct namespace.
 
### Adjust network policy
Make the accessing app flexible, this is because different apps will need access to the fb-av depending on the namespace. In the platform namespace, the user filestore requires access, in the saas namespace the editor will require access.

### Clean up old `deploy` folder and deploy steps
Since migrating to the new EKS cluster, we should no longer be deploying to the old ECR cluster.

Co-authored-by: Hellema Ibrahim <hellema.ibrahim@digital.justice.go.uk>
Co-authored-by: Brendan Butler <brendan.butler@digital.justice.gov.uk>

### CODE REVIEW

File: deploy-eks/fb-av-chart/templates/network_policy.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: allow-user-filestore-to-access-av
-  namespace: formbuilder-platform-{{ .Values.environmentName }}
+  name: allow-{{ .Values.accessing_app }}-to-access-av
+  namespace:  {{ .Values.namespace }}
```
Comment (brenetic) [inline deploy-eks/fb-av-chart/templates/network_policy.yaml]:
```suggestion
  namespace: {{ .Values.namespace }}
```

**Comment by brenetic** [comment]
this will need to be able to deploy to Live as well

**Comment by njseeto** [comment]
> this will need to be able to deploy to Live as well

Yep thought we could do that in a separate PR but am happy to add those steps and add to this PR if we want? I can also split out the `build_and_deploy_to_live` step, if we decide to do this

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**